sqlglot/dialects/clickhouse.py

```diff
@@ -44,8 +37,9 @@ class Parser(parser.Parser):
         FUNCTIONS = {
             **parser.Parser.FUNCTIONS,  # type: ignore
             "MAP": parse_var_map,
-            "QUANTILE": lambda args, params: exp.Quantile(this=params, quantile=args),
-            "QUANTILEIF": lambda args, params: exp.Quantile(this=params, quantile=args),
+            "QUANTILE": lambda params, args: exp.Quantile(this=args, quantile=params),
+            "QUANTILES": lambda params, args: exp.Quantiles(parameters=params, expressions=args),
+            "QUANTILEIF": lambda params, args: exp.QuantileIf(parameters=params, expressions=args),
```

Comment:
I changed the ordering of `args` and `params` here because it seems like the first wrapped csv is referred to as "parameters" in the [docs](https://clickhouse.com/docs/en/sql-reference/aggregate-functions/reference/quantiles/#quantiles)
<img width="868" alt="Screenshot 2023-01-06 at 11 56 00 PM" src="https://user-images.githubusercontent.com/46752250/211106805-256fb9ff-d453-4e10-a51b-373669b0df72.png">